Day-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,17 @@
 import React from 'react';
 import './Card.css';
+import CardBanner from './CardBanner';
+import CardContent from './CardContent';
+
+const CardContainer = () => {
+  return (
+    <div className="cardContainer">
+      <a href="https://www.reactjs.org" target="_blank" rel="noopener noreferrer" >
```

Comment:
Good job on implementing `rel="noopener noreferrer"`. I remember being really confused w/ the error message at first w/o it when first starting w/ React.